Deepthi fix 'Volunteer Agreement Confirmed' checkbox padding

deepthikannan opened the pull request:
# Description
![image](https://github.com/user-attachments/assets/90cfb13f-8316-4385-9a00-f9c6a9949ed9)
…

## Main changes explained:
- Updated AssignSetUpModal.jsx
- Added padding to the "Volunteer Agreement Confirmed" checkbox using inline styling to fix the misalignment issue.
- Refactored the checkbox component for better readability and maintainability.
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to dashboard→ user's profile (X)→ Under the profile picture, click on one of the buttons to get a popup titled "Assign X as Y"
6. verify that the "Volunteer Agreement Confirmed" checkbox is now properly aligned

![image](https://github.com/user-attachments/assets/3c0dcfce-b1d9-4bd0-b6b0-561d007cdf98)

Nahiyan-16 reviewed (changes requested):
Hi @deepthikannan, the checkbox is not centered. I see that you added left padding which helps align the element better but the checkbox and the label need to be vertically centered. 

![Screenshot 2024-08-10 141912](https://github.com/user-attachments/assets/8ec52125-6894-40f1-bd89-e4a84a2c8ed2)

JatinAgrawal94 reviewed (changes requested):
The alignment of checkbox needs to fixed slightly in vertical direction. Rest everything seems fine.
<img width="1512" alt="Screenshot 2024-08-10 at 7 14 35 PM" src="https://github.com/user-attachments/assets/a7c2ddfa-c90a-47d1-80fb-fa36f0450855">

aaryaneil reviewed (commented):
The changes mentioned in the PR look good to me but the link for the media folder is going beyond the modal.
<img width="1412" alt="Screenshot 2024-08-10 at 9 57 07 PM" src="https://github.com/user-attachments/assets/cb87859c-224a-47fe-893e-8dd9807f1601">

Aishwarya0898 reviewed (changes requested):
Hello @deepthikannan 

The checkbox is not aligned properly, this is how it is looking. Attaching screenshot for reference.

![PR Review #2554](https://github.com/user-attachments/assets/5e83f170-37c7-4ffb-86c3-78540136de08)

Pranay-ai reviewed (changes requested):
Hey @deepthikannan 
The PR is Working Fine for certain case but for the image below it fails to work.
![image](https://github.com/user-attachments/assets/21245434-f664-414a-a3d3-47bab2212b24)

The Link is Overlapping with the Background.

nikhilpittala16 reviewed (changes requested):
The alignment of the checkbox is not centered and the link is overflowing from the modal. it needs to be fixed.
![Screenshot (173)](https://github.com/user-attachments/assets/5d4d1ae9-7f3c-43a4-991c-edc118a973a6)

deepthikannan commented:
I've addressed the feedback of everyone who reviewed by fixing the alignment issue for the Volunteer Agreement Confirmed checkbox and the media folder link. Kindly please review the changes.

![image](https://github.com/user-attachments/assets/56f1e9fb-e962-4437-b71d-94b0b07c0643)

deepthikannan commented:
> Hi @deepthikannan, the checkbox is not centered. I see that you added left padding which helps align the element better but the checkbox and the label need to be vertically centered.
> 
> ![Screenshot 2024-08-10 141912](https://private-user-images.githubusercontent.com/71025942/356831812-8ec52125-6894-40f1-bd89-e4a84a2c8ed2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM4ODQxNDYsIm5iZiI6MTcyMzg4Mzg0NiwicGF0aCI6Ii83MTAyNTk0Mi8zNTY4MzE4MTItOGVjNTIxMjUtNjg5NC00MGYxLWJkODktZTRhODRhMmM4ZWQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE3VDA4MzcyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI1NTBjMjZhMGM5YWZkMWRkMGJkZWRmNTI5OTllOWVjOTU0NmNjNmM5MmMzZWI2NzI4MzE2ZmI1ZDBhMjg3NGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-G9Zf4eWqQ2yeCAn9HOZV8389B-Ur72zGo4QSCvWqoQ)

Hi I've made the necessary changes, kindly please review again. Thanks!

Pranay-ai reviewed (commented):
Hey @deepthikannan 
I have reviewed the changes, but the issues with media folder link still persists
![image](https://github.com/user-attachments/assets/46a983ed-68b6-4122-ad1e-4abe8ef4ecf6)

preethamdnr reviewed (changes requested):
Hello Deepthi,

I've reviewed your PR, and here are my observations:

The checkbox for "Volunteer Agreement Confirmed" is not perfectly aligned to the center. 
The link in the "Media Folder" section is overflowing beyond its container.

![image](https://github.com/user-attachments/assets/d3924b16-46dc-4989-b73d-a37cc6c5dc9d)

![image](https://github.com/user-attachments/assets/cafd665d-eca8-4df6-bed1-59dd877c8474)

RishithaMamidala reviewed (approved):
@deepthikannan 
The changes works as expected. The alignment of checkbox is prefect and the media link wraps when out of bounds.
![PR 2554](https://github.com/user-attachments/assets/1c4bdd5a-5d5e-4492-8711-347d1b2cfec5)

deepthikannan commented:
> Hey @deepthikannan I have reviewed the changes, but the issues with media folder link still persists ![image](https://private-user-images.githubusercontent.com/65746692/358866175-46a983ed-68b6-4122-ad1e-4abe8ef4ecf6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU2OTgwMDQsIm5iZiI6MTcyNTY5NzcwNCwicGF0aCI6Ii82NTc0NjY5Mi8zNTg4NjYxNzUtNDZhOTgzZWQtNjhiNi00MTIyLWFkMWUtNGFiZThlZjRlY2Y2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA3VDA4MjgyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIxODY3MmFiYzIwMWRlOGQ0NjE4MTBmOWI4Zjg3NjRiMmEyNThlZDBjMjgwMDRmZDJjYjBmM2VmMDMzMzg0NTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QsWkohoeN54axguU84CGxjLLI1GA39mCJcL-IyoCAMo)

Hi @Pranay-ai could you kindly check again? i've attached my result here.
![image](https://github.com/user-attachments/assets/328f020c-1cae-4423-8732-39458a987d0f)

deepthikannan commented:
> Hello Deepthi,
> 
> I've reviewed your PR, and here are my observations:
> 
> The checkbox for "Volunteer Agreement Confirmed" is not perfectly aligned to the center. The link in the "Media Folder" section is overflowing beyond its container.
> 
> ![image](https://private-user-images.githubusercontent.com/57593597/360591654-d3924b16-46dc-4989-b73d-a37cc6c5dc9d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU2OTgwMDQsIm5iZiI6MTcyNTY5NzcwNCwicGF0aCI6Ii81NzU5MzU5Ny8zNjA1OTE2NTQtZDM5MjRiMTYtNDZkYy00OTg5LWI3M2QtYTM3Y2M2YzVkYzlkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA3VDA4MjgyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE5ZjRkMWQ0MDNlMmJkMmRmYmM4NTFmMmI2ZGFhY2FhZDFlMzg1OGFjZjkzZjc1OTUzNjY1ZDM3OWYyZTcyZjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Y-Mb7qqZsDh4Oa_wdFNfd4BgKCmDTo5EZv2SA7AtQbQ)
> 
> ![image](https://private-user-images.githubusercontent.com/57593597/360591686-cafd665d-eca8-4df6-bed1-59dd877c8474.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU2OTgwMDQsIm5iZiI6MTcyNTY5NzcwNCwicGF0aCI6Ii81NzU5MzU5Ny8zNjA1OTE2ODYtY2FmZDY2NWQtZWNhOC00ZGY2LWJlZDEtNTlkZDg3N2M4NDc0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA3VDA4MjgyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVmMzkwYTYxNjE5ODY2ZGU3MzVhMGFjNDMzNTE0NWU0ZDkwZTMwNjFiZTkwOGEwNTQyZjlkMmNmNjkwYTk2MmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ERWhtI79FyRxyFiNAznL4w4Y_GY1ZGLOKXqAqva9MDg)

Hi @preethamdnr , could you kindly check again by logging in into owner account, i've attached my result here.
![image](https://github.com/user-attachments/assets/c04cb1b1-a19f-4ef5-a5df-e4eb92fa5e03)

AJAYINAVOLU reviewed (approved):
Changes are working as expected.

<img width="938" alt="image" src="https://github.com/user-attachments/assets/c8e18a63-2521-4b7a-ac13-e30e2d9a8b29">

Thank You

Abhinav-Ankur786 reviewed (approved):
Box is in center now as per PR

<img width="1477" alt="PR #2554" src="https://github.com/user-attachments/assets/64552d79-3e9d-4613-9722-5c3f035b8aed">

navya9989 commented:
Hi @deepthikannan , I have tested this PR and i could see that the check box need to be aligned slightly to the centre. Here is the Picture for reference. 
![PR #2554](https://github.com/user-attachments/assets/192d4e0f-ccd5-4adf-ae50-24f988c4c705)
